Makefile

```diff
@@ -14,7 +14,7 @@ install: install-node install-python .git/hooks/pre-commit
 
 lint:
 	npm run lint
-	find . -name '*.py' -not -path '**/.venv/*' | xargs poetry run flake8
+	find . -name '*.py' -not -path '**/.venv/*' | xargs poetry run flake8 --ignore=E501
```

Comment:
E501 for larger than life lines? Can be ignored? Where in the codebase aside from unittests are we running python?

Reply:
Also it can be set up in the ini file to ignore as well. Just like I dunno you think flake8 lint just for apigee unit tests (I guess through the whole framework py) weird? I've always thought it was tbh.

Reply:
This is code from the core template which is failing linting rules from the core template :/